### PR TITLE
#105: Add path prefix too all routes (closes #105)

### LINKED ISSUE #105: Add path prefix too all routes
## requirements
- the user should be allowed to add a path prefix to all routes
- the prefix should be configurable from configuration or environment variable

## specs
{optional: describe technical specs to implement this feature, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by calippo**
Closes #105

### tests performed
- tested locally

### CODE REVIEW

File: serverAkkaHttp/src/main/resources/reference.conf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+wiro {
+  routes-prefix = ${?WIRO_ROUTES_PREFIX}
+  ciao = "lol"
```
Comment (danielegallingani) [inline serverAkkaHttp/src/main/resources/reference.conf]: leftover?

**Review by danielegallingani — DISMISSED** [review]
<3 1 comment, could you also release a new version?

File: serverAkkaHttp/src/main/resources/reference.conf (hunk 1 above)
Comment (calippo) [reply serverAkkaHttp/src/main/resources/reference.conf]: whops

Comment (calippo) [reply serverAkkaHttp/src/main/resources/reference.conf]: whops

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.6.6-SNAPSHOT"
+version in ThisBuild := "0.6.6"
```
Comment (gabro) [inline version.sbt]: you shouldn't manually change this.

Just do `> release` in sbt

Comment (calippo) [reply version.sbt]: That change was produced by `> release` command

**Pull request merged**